src/hydra/dataProvider.ts

```diff
@@ -662,14 +657,15 @@ export default (
       apiSchema
         ? Promise.resolve({ data: apiSchema })
         : apiDocumentationParser(entrypoint)
-            .then(({ api, customRoutes = [] }) => {
```

Comment:
Why removing the `customRoutes`?

Reply:
good catch, i did this by error because of typescript infering the wrong type on this one and raising an error (`Property 'customRoutes' does not exist on type '{ api: Api; response: Response; status: number; }'.ts(2339)`) 

I revert it to the original with an any type for the moment